src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -370,6 +371,41 @@ def get_gpu_instance_profile(self) -> Union[str, None]:
         # this parameter does not need validation
         return gpu_instance_profile
 
+    def get_message_of_the_day(self) -> Union[str, None]:
+        """Obtain the value of message_of_the_day.
+
+        :return: string or None
+        """
+        # read the original value passed by the command
+        message_of_the_day = None
+        message_of_the_day_file_path = self.raw_param.get("message_of_the_day")
+
+        if message_of_the_day_file_path:
+            if not os.path.isfile(message_of_the_day_file_path):
+                raise InvalidArgumentValueError(
```

Comment:
Missing unit test coverage for this exception. See pipeline [log](https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=52206529&view=logs&j=1c778e8e-596b-5a0a-22c2-460364725d7c&t=3e82742e-7cc5-5f7d-8ec2-cd9417b56eb7&l=386).